Installation.md

```diff
@@ -586,8 +586,8 @@ See [[Installing Chocolatey Behind a Proxy Server|Proxy-Settings-for-Chocolatey#
   * Copy/Move over the items from the old lib/bin directory.
   * Delete your old install directory.
 
-**NOTE**: There is one really important consideration when installing Chocolatey to a non-default location - Chocolatey only locks down the permissions to Admins when installed to the default location.
-If you are installing to another location, you will need to handle this yourself.
+**NOTE**: There is one really important consideration when installing Chocolatey to a non-default location: Chocolatey only locks down the permissions to Admins when installed to the default location, which is <em>`<SYSTEM_DRIVE>`</em>`:\ProgramData\Chocolatey`.

```

Comment:
Close - `<ALLUSERSPROFILE>\Chocolatey` sometimes also known at `<PROGRAMDATA>`

Reply:
Good point, thanks for your thoughts!

Would stating both variants be OK? If only to spare the reader the extra step of figuring out where `%ALLUSERSPROFILE%` is?

> Chocolatey only locks down the permissions to Admins when installed to the default location `%ALLUSERSPROFILE%\Chocolatey`, which means the same thing as `%PROGRAMDATA%\Chocolatey` and `%SystemDrive%:\ProgramData\Chocolatey`.

(The code indeed [checks both variants](https://github.com/chocolatey/choco/blob/1f8024c102cebe0cf2628cdab4e322416376036c/nuget/chocolatey/tools/chocolateysetup.psm1#L197-L202).)

Reply:
I think that is fine. You could even leave out ALLUSERSPROFILE and prefer PROGRAMDATA. The only place that makes a difference is Windows 2003 and we are slowly weaning off of that system as it is fully out of support by Microsoft.

Reply:
Sounds reasonable. Done.